### ISSUE TITLE
Can't force specific debugger cmd

### ISSUE THREAD

**Issue by fdev31** [python-mode/python-mode#795]
One can't set a custom command for the debugger, eg. in the configuration file:
```
let g:pymode_breakpoint_cmd = "import rpdb2; rpdb2.start_embedded_debugger('xxx')"
```

To allow this, I moved the ``PymodePython <<EOF .... EOF`` code, in **autoload/pymode/breakpoint.vim**, just after the
```
        if g:pymode_python == 'disable'
            return
        endif
```

So the code is not "detecting" debugger if some is already set by the user...

**Comment by fdev31** [comment]
I don't do pull requests because I'm not used to do any vimscript and probably don't know what I'm doing... the previous code was breaking the startup somehow. Here is a version which sounds better to me:

```diff
diff --git a/autoload/pymode/breakpoint.vim b/autoload/pymode/breakpoint.vim
index c3189aa..6f26aaf 100644
--- a/autoload/pymode/breakpoint.vim
+++ b/autoload/pymode/breakpoint.vim
@@ -5,7 +5,7 @@ fun! pymode#breakpoint#init() "{{{
     endif
 
     if g:pymode_breakpoint_cmd == ''
-        let g:pymode_breakpoint_cmd = 'import pdb; pdb.set_trace()  # XXX BREAKPOINT'
+"        let g:pymode_breakpoint_cmd = 'import pdb; pdb.set_trace()  # XXX BREAKPOINT'
 
         if g:pymode_python == 'disable'
             return
@@ -16,14 +16,17 @@ fun! pymode#breakpoint#init() "{{{
         PymodePython << EOF
 
 from imp import find_module
+import vim
 
-for module in ('wdb', 'pudb', 'ipdb'):
-    try:
-        find_module(module)
-        vim.command('let g:pymode_breakpoint_cmd = "import %s; %s.set_trace()  # XXX BREAKPOINT"' % (module, module))
-        break
-    except ImportError:
-        continue
+if not vim.eval("g:pymode_breakpoint_cmd"):
+
+    for module in ('rpdb2', 'wdb', 'pudb', 'ipdb'):
+        try:
+            find_module(module)
+        except ImportError:
+            continue
+        else:
+            vim.command('let g:pymode_breakpoint_cmd = "import %s; %s.set_trace()  # XXX BREAKPOINT"' % (module, module))
 
 EOF

```

**Comment by alphaCTzo7G** [comment]
Perhaps we  should rewrite/extend this plugin for python for python-mode in python.. so more people can contribute..

**Comment by fmv1992** [comment]
Actually I've been doing this (developing this plugin) alone for a while.

I have my own fork of it which I intend to merge into this from time to time (I
have writing permissions to this fork) but the module redesign proved to be
more drastic than initially supposed. And as the decisions of the development
were made on my own I was quite inhibited to push them here.

I opted to keep this as is, as good or as bad as it is.

Meanwhile issues have been mounting in this fork and I've been solving/developing whathever features I like.

Comparing both forks I think the major redesign was to move those 'static' python files to
submodules and them symlink them. This was a response to their periodical
obsolescence (they are copy pasted from other modules).

Anyways I would be delighted to redirect you to my own fork and get some actual
feedback (other than my own judgement) and then merge again the development of
all forks into a single place (here).

A couple of issues found by other people are already fixed there.

Link: <https://github.com/fmv1992/python-mode>

I'm eager to know what you think.

**Please use the dev branch.**

**Comment by fdev31** [comment]
How is it supposed to work?
I've been using in my configuration:
let g:pymode_breakpoint_cmd = 'BLAH'
but it seems to be overridden...

**Comment by fmv1992** [comment]
Fixed on commit 65c68ea.

I'm sorry for the delay.

Best,
